playground/relay-kit/usdc-transfer-4337-counterfactual.ts

```diff
@@ -89,27 +93,19 @@ async function main() {
   const safeOperation = await safe4337Pack.createTransaction({
     transactions,
     options: {
-      validAfter: timestamp,
-      validUntil: timestamp + 60000
+      validAfter: timestamp - 60_000,
+      validUntil: timestamp + 60_000
     }
   })
 
-  // 3) Estimate SafeOperation fee
-  const feeEstimator = new PimlicoFeeEstimator()
```

Comment:
If we don't add the feeEstimator to the `createTransaction` it won't be possible to select a custom feeEstimator